### ISSUE TITLE
pojo installation support

### ISSUE THREAD

**Issue by jasonkuhrt** [littlebits/cloud-client-api-http#5]
Need a simple script with public URI that people of all skill levels can include in html pages and immediately use. Most likely will register a global object called `littleBitsAPI` or similar, a simple example usage would be such as https://github.com/littlebits/cloud-api-lessons/issues/4

- [x] Committed dist
- [x] Committed dist-min
- [x] littleBits cdn hosting of dist + dist-min

**Comment by stresslimit** [comment]
Now accessible in at least temporary location: 

```html
<script src="http://apps.littlebitscloud.cc/api-http/littleBitsAPI.js"></script>
<script src="http://apps.littlebitscloud.cc/api-http/littleBitsAPI.min.js"></script>
```
Closed by 01b08594

**Comment by stresslimit** [comment]
@jasonkuhrt @logand
